src/navigation/nav_bringup/config/slam_toolbox_params.yaml

```diff
@@ -31,7 +31,7 @@ slam_toolbox_node:
     max_laser_range: 25.0  # Maximum laser range to use for 2D occupancy map rastering
     minimum_time_interval: 0.1  # The minimum duration of time between scans to be processed in synchronous mode
     transform_timeout: 0.5  # TF timeout for looking up transforms
-    tf_buffer_duration: 1.0  # Duration to store TF messages for lookup. Set high if running offline at multiple times speed in synchronous mode
+    tf_buffer_duration: 2.0  # Duration to store TF messages for lookup. Set high if running offline at multiple times speed in synchronous mode
```

Comment:
Note here, this seems to be a sensitive param. Deserves more looking into its affect while live. Affects on recorded data can be quite substantial - possibly recording timing related?